test/integration/settings_test.rb

```diff
@@ -111,6 +111,48 @@ def mfa_key
     assert page.has_content? "You have not yet enabled multi-factor authentication."
   end
 
+  test "Clicking MFA continue button without copying recovery codes creates confirm popup" do
```

Comment:
Looks like this test is flaky https://github.com/rubygems/rubygems.org/runs/6942326906?check_suite_focus=true

Reply:
sleep may work but it would not definitive solution and be both overfit and underfit depending on test run. capybara has method which goes something like wait for element to appear. You should use that.

Reply:
https://github.com/rubygems/rubygems.org/pull/3124